### PR TITLE
Add init script for openwrt.

### PR DESCRIPTION
**Pull request by tomwilkie**
Signed-off-by: Tom Wilkie <tom@grafana.com>

### CODE REVIEW

File: examples/openwrt-init.d/dnsmasq_exporter [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/bin/sh /etc/rc.common
+
+START=99
+
+USE_PROCD=1
+PROG="/usr/bin/dnsmasq_exporter"
+OPTIONS="-leases_path=/tmp/dchp.leases"
```
Comment (stapelberg) [inline examples/openwrt-init.d/dnsmasq_exporter:7]: typo: dhcp

Is this the default path to the leases file on OpenWrt?

**Comment by stapelberg** [comment]
Shouldn’t this go into the OpenWrt packaging repos instead?
Seems kinda weird to maintain these in an upstream repository.

File: examples/openwrt-init.d/dnsmasq_exporter (hunk 1 above)
Comment (tomwilkie) [reply examples/openwrt-init.d/dnsmasq_exporter]: Well spotter! Fixes.

And yeah, this seems like the default place for the leases file.

**Comment by tomwilkie** [comment]
> Shouldn’t this go into the OpenWrt packaging repos instead?
> Seems kinda weird to maintain these in an upstream repository.

These isn't upstream packaging for this yet.  I thought the init script could go in this repo as you have the systemd unit definition here too.  If I get a chance I'll try and build some packaging, but can't promise anything today!

**Comment by stapelberg** [comment]
Gotcha. Okay, let’s put it here and move it later if a package appears in OpenWrt :)

**Pull request merged**